### PR TITLE
Proofreading.

### PR DESCRIPTION
**Pull request by MariuszXC**
Some language cleanup.
References to files modified to reflect the directory structure as delivered to the user.  Had to sacrifice navigation in source tree in order. Don't see an easy way around it, unless splitting main README.md into short generic intro (with in-tree working links) and e.g. README-plugin.md meant for asset inclusion (and perhaps a similar README-server.md for server-side docs).

### CODE REVIEW

**Comment by hbeni** [comment]
> References to files modified to reflect the directory structure as delivered to the user.  Had to sacrifice navigation in source tree in order. Don't see an easy way around it.

There is an easy way around it: I include this replacements dynamically in the build scripts.
I would really want to let the links intact for easy github browsing.
Good catch, i will add that - can you please revert those links?

**Comment by MariuszXC** [comment]
Links reverted to original state.
Two point to server part of the tree. Probably candidates for dynamic removal in build?

**Review by hbeni — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -49,7 +47,7 @@ Setup requirements
 
 Installation
 -----------------------
-The release ZIP contains all binary plugins for all supported operating systems in the `mumble-plugin` folder:
```
Comment (hbeni) [inline README.md:52]: Spaces removed? (Markdown syntax)

File: README.md [hunk 2]
```diff
@@ -98,12 +96,12 @@ You are ready for radio usage! Some client needs to supply information to the pl
...
 
-Note that frequencies can be arbitary strings. That said, all participating clients must share a common definition of "frequency", this should be the physical radio wave frequency in MHz and not the "channel" (esp. with 8.3 channels spacing).  
+Note that frequencies can be arbitary strings. That said, all participating clients must share a common definition of "frequency", this should be the physical radio wave frequency in MHz and not the "channel" (esp. with 8.3 channels spacing).
 Also note that callsigns and frequencies are not allowed to contain the comma symbol (`,`). Decimal point symbol has always to be a point (`.`).
 
 
 ### RadioGUI
-FGCom-mumble releases ship with a cross-plattform java application that implements most of the UDP protocol and thus can be used not only for testing purposes, but also real operations without the need for another client.  
+FGCom-mumble releases ship with a cross-plattform java application that implements most of the UDP protocol and thus can be used not only for testing purposes, but also real operations without the need for another client.
```
Comment (hbeni) [inline README.md:106]: Spaces removed? (Markdown syntax)

File: README.md [hunk 3]
```diff
@@ -115,7 +113,7 @@ For details on how this can be done, look at RadioGUI's readme.
 Just add and activate the [FGFS-addon](client/fgfs-addon/Readme.md) in your launcher.
 
 The FGFS protocol file will handle old 25kHz as well as newer 8.3kHz radios.
-After starting flightgear, you can use your radio stack like with FGCom (default is *space* to talk on COM1 and *shift+space* for COM2).  
+After starting flightgear, you can use your radio stack like with FGCom (default is *space* to talk on COM1 and *shift+space* for COM2).
```
Comment (hbeni) [inline README.md:118]: Spaces removed? (Markdown syntax)

File: README.md [hunk 4]
```diff
@@ -127,7 +125,7 @@ Since ATC-Pie v1.7.1 FGCom-mumble is supported out of the box.
 ### OpenRadar specific
 Currently, OpenRadar just supports one Radio per UDP port. In case you want several Radios (which is likely), you need to invoke several dedicated mumble processes. This will give you separate FGCom-mumble plugin instances listening on different ports, and in OpenRadar you can thus specify that ports.
 
-For better FGCom-mumble support, [patches are already pending](https://sourceforge.net/p/openradar/tickets/) and there is a [binary package available](http://fgcom.hallinger.org/OpenRadar_fgcom-mumble.jar).  
+For better FGCom-mumble support, [patches are already pending](https://sourceforge.net/p/openradar/tickets/) and there is a [binary package available](http://fgcom.hallinger.org/OpenRadar_fgcom-mumble.jar).
```
Comment (hbeni) [inline README.md:130]: Spaces removed? (Markdown syntax)

File: README.md [hunk 5]
```diff
@@ -150,7 +148,7 @@ To record an ATIS sample, you need to:
 
 Regular recordings have a serverside limit of 120 seconds by default.
 
-Note: Chances are good that your ATC client does all this for you and you just need to push some "Record ATIS" button.  
+Note: Chances are good that your ATC client does all this for you and you just need to push some "Record ATIS" button.
```
Comment (hbeni) [inline README.md:153]: Spaces removed? (Markdown syntax)

File: README.md [hunk 6]
```diff
@@ -200,13 +199,13 @@ When you cannot hear other pilots or are unable to transmit on the radios, you c
...
 - Look at the murmur server log for possible dropped plugin messages (look for the string `Dropping plugin message`), they may cause out of sync state. Reasons can be:
-  - the setting *`pluginmessagelimit`* in `murmur.ini`  may be too restrictive.
+  - the setting *`pluginmessagelimit`* in `murmur.ini` may be too restrictive.
   - a bug in the plugin-io code: The plugin is expected to work well with default settings, so dropped messages may indicate a plugin bug; especially if they appear rapidly over a longer time.
 
 
 Compiling the plugin
 ======================
-The FGCom-mumble client plugin needs to be in binary form. If you want to use the latest code from github, you can compile yourself. The makefile is tailored to be used mainly on linux, but can be used in windows and macOS too.  
+The FGCom-mumble client plugin needs to be in binary form. If you want to use the latest code from github, you can compile yourself. The makefile is tailored to be used mainly on linux, but can be used in windows and macOS too.
```
Comment (hbeni) [inline README.md:210]: This one is good :+1:

File: README.md [hunk 7]
```diff
@@ -231,7 +230,7 @@ Other interesting compile targets:
 
 Windows native build
 --------------------
-The makefile works well on Windows with cygwin64 with mingw32.  
+The makefile works well on Windows with cygwin64 with mingw32.
```
Comment (hbeni) [inline README.md:235]: ok

File: Readme.architecture.md [hunk 8]
```diff
@@ -4,47 +4,47 @@ FGCom-mumble - a flightsim radio simulation framework based on mumble
...
 
 The FGCom mumble client plugin
 ------------------------------
 This is the central cornerstone of the system. The plugin handles sending/receiving of the radio broadcasts from players and also bots.
-For this to work, the plugins (and bots) share positional and frequency information trough mumbles builtin plugin data send interface. When the user connects, the plugin will start to share this data automatically and collect the information from already connected clients. This way, each local plugin has always a recent knowledge of the other users.
+For this to work, the plugins (and bots) share positional and frequency information trough mumble's builtin plugin data send interface. When the user connects, the plugin will start to share this data automatically and collect the information from already connected clients. This way, each local plugin has always a recent knowledge of the other users.
 
-For the plugin to commence operations, a special channel `fgcom-mumble` must be joined (so other communication stays unaffected and the plugin can remain active for normal talking elsewhere).  
-If a users fgcom mumble plugin then receives a new audio transmission, it will:
```
Comment (hbeni) [inline Readme.architecture.md:16]: Spaces removed? (Markdown syntax)

File: Readme.architecture.md [hunk 9]
```diff
@@ -4,47 +4,47 @@ FGCom-mumble - a flightsim radio simulation framework based on mumble
...
-- look at its data to get the frequency of the current transmission
+For the plugin to commence operations, a special channel `fgcom-mumble` must be joined on a mumble server (so other communication stays unaffected and the plugin can remain active for normal talking elsewhere).
+If a user's fgcom mumble plugin then receives a new audio transmission, it will:
+- look at transmission's data to get its frequency
 - look at the location of the sender
-- look at current transmission state of the local user
-Then, if the frequency is currently tuned AND the sender was in radio-range AND the current user is not transmitting himself on the frequency in question, the received transmision will be played; otherwise discarded.
```
Comment (hbeni) [inline Readme.architecture.md:20]: Spaces removed? (Markdown syntax)

File: Readme.architecture.md [hunk 10]
```diff
@@ -4,47 +4,47 @@ FGCom-mumble - a flightsim radio simulation framework based on mumble
...
 
 Also, the plugin can send information via an UDP interface to third party software at 10Hz on UDP.
 
-Details are too explained in the `plugin-spec.md` file.
+Protocol and interface details are explained in the `plugin-spec.md` file.
 
 
 ### Flightgear integration
-To send data to the plugin, flightgear must be startet with property-tree synchronization trough a generic protocol.  
+To send data to the plugin, flightgear must be started with property-tree synchronization trough a generic protocol.
```
Comment (hbeni) [inline Readme.architecture.md:40]: Spaces removed? (Markdown syntax)

File: Readme.architecture.md [hunk 11]
```diff
@@ -55,21 +55,21 @@ ATC support
 ATC clients can connect using the old FGCom UDP protocol or using the newer one.
 
 ## Position, `HGT=` setting / Antenna height
-In either case, it is important to set a valid position and altitude. Altitude is the main range limiting factor in VHF radio ooperations, for example 1m heigth gives about 3.6km range until your transmission hits the earths surface. it is advised that you set the altitude to the antenna tip height above surface (so 8m building+2m Antenna gives 10m=32.8ft: `HGT=32.8`.
+In either case, it is important to set a valid position and altitude. Altitude is the main factor limiting range in VHF radio operations, for example 1m height gives about 3.6km range until your transmission hits the earth's surface. It is advised that you set the altitude to the antenna tip height above surface (so 8m building+2m Antenna gives 10m=32.8ft: `HGT=32.8`.
 
 ### Land lines
-You can establish virtual land lines by adding a new "virtual radio" with a special custom frequency starting with `PHONE` like "PHONE-EDDM-TWR". Such connections are not subject to radio signal quality or range and allow for full-duplex operation.
```
Comment (hbeni) [inline Readme.architecture.md:61]: Spaces removed? (Markdown syntax)

File: Readme.architecture.md [hunk 12]
```diff
@@ -55,21 +55,21 @@ ATC support
...
 
 ## Position, `HGT=` setting / Antenna height
-In either case, it is important to set a valid position and altitude. Altitude is the main range limiting factor in VHF radio ooperations, for example 1m heigth gives about 3.6km range until your transmission hits the earths surface. it is advised that you set the altitude to the antenna tip height above surface (so 8m building+2m Antenna gives 10m=32.8ft: `HGT=32.8`.
+In either case, it is important to set a valid position and altitude. Altitude is the main factor limiting range in VHF radio operations, for example 1m height gives about 3.6km range until your transmission hits the earth's surface. It is advised that you set the altitude to the antenna tip height above surface (so 8m building+2m Antenna gives 10m=32.8ft: `HGT=32.8`.
 
 ### Land lines
-You can establish virtual land lines by adding a new "virtual radio" with a special custom frequency starting with `PHONE` like "PHONE-EDDM-TWR". Such connections are not subject to radio signal quality or range and allow for full-duplex operation.  
-Volume settings and operational state of the simulated phone is applied, however.
```
Comment (hbeni) [inline Readme.architecture.md:62]: Spaces removed? (Markdown syntax)

File: Readme.architecture.md [hunk 13]
```diff
@@ -79,8 +79,8 @@ The following features should be implemented:
  3. *Silent recording to other pilots:* It is not desirable that the recording is transmitted to nearby pilots instantly.
  4. *Easy recording:* No special software should be needed to conduct recording. It should be done via the in-place infrastructure.
 
-This can be easily supported trough a special `radio-recorder` bot that will listen for incoming record requests over the mumble plugin API.
```
Comment (hbeni) [inline Readme.architecture.md:82]: Spaces removed? (Markdown syntax)

File: Readme.architecture.md [hunk 14]
```diff
@@ -79,8 +79,8 @@ The following features should be implemented:
  3. *Silent recording to other pilots:* It is not desirable that the recording is transmitted to nearby pilots instantly.
  4. *Easy recording:* No special software should be needed to conduct recording. It should be done via the in-place infrastructure.
 
-This can be easily supported trough a special `radio-recorder` bot that will listen for incoming record requests over the mumble plugin API.  
-Recording has to be done using a special frequency like `RECORD_<tgtFreq>`. The other pilots will not hear the recording, because they can't tune into this frequency. Just the `radio-recorder` bot will monitor all frequencies starting with `RECORD_`-prefix and record anything that comes in. As the target frequency and location must be set from the client (atc-instance in this case), the bot will receive anything that is needed to get frequency and location of the broadcast.
```
Comment (hbeni) [inline Readme.architecture.md:83]: Spaces removed? (Markdown syntax)

File: Readme.architecture.md [hunk 15]
```diff
@@ -89,5 +89,5 @@ Just invoke a `fgcom-radio-playback` bot with the radio station audio program fi
 
 Status webpage
 ----------------------------
-For user convinience there is also a webpage that shows the current status.
```
Comment (hbeni) [inline Readme.architecture.md:92]: Spaces removed? (Markdown syntax)

File: client/fgfs-addon/Readme.md [hunk 16]
```diff
@@ -11,13 +11,13 @@ Instead of using this addon, you can also invoke the protocol manually:
 
 Installation
 ------------
-After unzipping the FGCom-mumble release package, you just need to add the `fgfs-addon` folder to your launchers *Add-ons* module list.
```
Comment (hbeni) [inline client/fgfs-addon/Readme.md:14]: Spaces removed? (Markdown syntax)

File: client/fgfs-addon/Readme.md [hunk 17]
```diff
@@ -26,5 +26,5 @@ After starting flightgear, you can use your radio stack like with FGCom (default
 
 Configuration
 ----------------------------
-If you wish to adjust the parameters, you can access them via the new *Multiplayer* menu entry. This is usually not needed except you are running several mumble instances or mumble not on the same computer as FlightGear.
```
Comment (hbeni) [inline client/fgfs-addon/Readme.md:29]: Spaces removed? (Markdown syntax)

File: client/radioGUI/Readme.RadioGUI.md [hunk 18]
```diff
@@ -20,7 +20,7 @@ You can run several instances of the application at once to have different clien
 
 SimConnect support (MSFS2020)
 -----------------------------
-RadioGUI can connect to a SimConnect compatible simulator like MFSF2020.
```
Comment (hbeni) [inline client/radioGUI/Readme.RadioGUI.md:23]: Spaces removed? (Markdown syntax)

**Review by hbeni — CHANGES_REQUESTED** [review]

**Comment by hbeni** [comment]
> Links reverted to original state.
> Two point to server part of the tree. Probably candidates for dynamic removal in build?

No, i want to keep them mentioned. The build script does remove the links and add "(not included)" in those cases.

File: README.md [hunk 19]
```diff
@@ -98,12 +96,12 @@ You are ready for radio usage! Some client needs to supply information to the pl
 ### Generic compatibility
 The plugin aims to be compatible to the legacy fgcom-standalone protocol, so vey much all halfway recent fgfs instances, ATC clients and aircraft should handle it out of the box at least with COM1.
 
-Note that frequencies can be arbitary strings. That said, all participating clients must share a common definition of "frequency", this should be the physical radio wave frequency in MHz and not the "channel" (esp. with 8.3 channels spacing).
```
[the remark this answers is not in the document]
Comment (MariuszXC) [reply README.md:101]: Spaces were an oversight on my part. Keep'em.

File: Readme.architecture.md [hunk 20]
```diff
@@ -4,47 +4,47 @@ FGCom-mumble - a flightsim radio simulation framework based on mumble
...
-Details are too explained in the `plugin-spec.md` file.
+Protocol and interface details are explained in the `plugin-spec.md` file.
 
 
 ### Flightgear integration
-To send data to the plugin, flightgear must be startet with property-tree synchronization trough a generic protocol.  
+To send data to the plugin, flightgear must be started with property-tree synchronization trough a generic protocol.
 We strongly advise to use the new protocol format.
-The [new protocol xml-file](client/fgfs/Protocol/fgcom-mumble.xml) is supplied in the source tree and documented (`client/fgfs/Protocol/fgcom-mumble.xml`).
+The [new protocol xml-file](fgfs-addon/FGData/Protocol/fgcom-mumble.xml) is supplied in the source tree and documented (`client/fgfs-addon/FGData/Protocol/fgcom-mumble.xml`).
```
[the remark this answers is not in the document]
Comment (MariuszXC) [reply Readme.architecture.md:42]: As you said, two pair of eyes are better than one :)

**Comment by hbeni** [comment]
Would be easier to merge if the spaces are out.
I can also merge and revert partly, no problem , but i will be not home this weekend

File: README.md (hunk 4 above)
Comment (MariuszXC) [reply README.md:130]: Keep'em

**Comment by MariuszXC** [comment]
OK, sorry for the mess. I'll try to prepare one more pull request. I hope I'll repair all.

**Comment by MariuszXC** [comment]
Found two more places. Hopefully last. 
If you accept, this may be closed?

File: Readme.architecture.md [hunk 21]
```diff
@@ -65,11 +65,11 @@ A good practice may be the syntax `PHONE:[ICAO]:[POS](:[LINE])`, like `PHONE:EDD
...
 
 ### ATIS playback
-A `fgcom-radio-playback`-bot can connect to the mumble server. For that to work properly, he will be called with the needed information: frequency to send on and its location. This information will be broadcasted over the mumble plugin interface, so the other mumble pugins of the pilots can pick it up. From then on, the bot behaves as an ordinary radio client from the view of the plugins.
-The bot will read a specified audio file and braodcast it on the selected frequency, until either he is killed or the audio file is deleted (then he kills himself).
+A `fgcom-radio-playback`-bot can connect to the mumble server. For that to work properly, it will be called with the needed information: frequency to transmit on and its location. This information will be broadcast over the mumble plugin interface, so the other mumble pugins of the pilots can pick it up. From then on, the bot behaves as an ordinary radio client from the view of the plugins.
+The bot will read a specified audio file and broadcast it on the selected frequency, until either it is killed or the audio file is deleted (then the bot terminates).
```
Comment (hbeni) [inline Readme.architecture.md:73]: First i was inclined to "hang itself" but that would had indicated a dangling process (kidding)

Honestly, thank you for this, I didn't want to use violent language here but was not able to come up with something better :) :+1:

**Review by hbeni — APPROVED** [review]
Thank you very much for this! Much apprechiated, and sorry for my nagging :)

**Pull request merged**